homeassistant/components/logbook/__init__.py

```diff
@@ -458,7 +467,22 @@ def yield_events(query):
             .filter((Events.time_fired > start_day) & (Events.time_fired < end_day))
         )
 
-        if entity_ids:
+        if entity_matches_only and entity_ids and len(entity_ids) == 1:
```

Comment:
Should we raise an error if multiple entity IDs are passed in with this option ?

Reply:
Just realized that the api only takes one anyways.  This can be refactored a bit to reduce the complexity